src/Model/ValueObject/Collection/Collection.php

```diff
@@ -41,6 +41,21 @@ public function with($value) // @phpstan-ignore-line III-5812 Can be fixed once
         return new static(...$values);
     }
 
+    /**
+     * @return static
+     */
+    public function without($value) // @phpstan-ignore-line
```

Comment:
```suggestion
    public function without(Image $value) // @phpstan-ignore-line
```
Can this be Image type?

Reply:
No, because this `Collection` is generic and unfortunately there are no generics in PHP.

Reply:
never mind, I thought I was looking in the specific ImageColection.